llvm/test/TableGen/trydecode-emission3.td

```diff
@@ -1,4 +1,4 @@
-// RUN: llvm-tblgen -gen-disassembler -I %p/../../include %s | FileCheck %s
+ // RUN: llvm-tblgen -gen-disassembler --num-to-skip-size=3 -I %p/../../include %s  | FileCheck %s
```

Comment:
Extra space at col 1 :)

Should test `=2` as well (other tests, too).

Reply:
Done. Only for tests that actually test the decoder table contents.